packages/commonwealth/client/scripts/views/pages/search/helpers.tsx

```diff
@@ -20,16 +20,16 @@ const getDiscussionResult = (thread, searchTerm, setRoute) => {
   const proposalId = thread.proposalid;
   const chain = thread.chain;
 
+  const handleClick = () => {
+    setRoute(`/${thread.chain}/discussion/${proposalId}`);
```

Comment:
if you will click `search all communities?` and then click thread that is outside of the community you are currently in, it will fail. 

https://user-images.githubusercontent.com/14819225/232078366-421a7ce5-0ea4-49d7-adec-ae574127abec.mp4




I am pretty sure that this will work
```
    setRoute(`/discussion/${proposalId}`, {}, thread.chain);
```